core/src/main/java/org/apache/iceberg/CachingCatalog.java

```diff
@@ -164,22 +164,19 @@ public Table loadTable(TableIdentifier ident) {
   @Override
   public boolean dropTable(TableIdentifier ident, boolean purge) {
     boolean dropped = catalog.dropTable(ident, purge);
-    invalidate(ident);
+    invalidateTable(ident);
     return dropped;
   }
 
   @Override
   public void renameTable(TableIdentifier from, TableIdentifier to) {
     catalog.renameTable(from, to);
-    invalidate(from);
+    invalidateTable(from);
   }
 
   @Override
   public void invalidateTable(TableIdentifier ident) {
-    invalidate(ident);
-  }
-
-  private void invalidate(TableIdentifier ident) {
```

Comment:
The private `invalidate(ident)` method wasn't doing anything other than calling the public `invalidateTable(ident)` method. I got rid of it.